lib/open_api_spex/plug/redoc_ui.ex

```diff
@@ -0,0 +1,58 @@
+defmodule OpenApiSpex.Plug.RedocUI do
+  @moduledoc """
+  Module plug that serves RedocUI.
```

Comment:
Can you add code examples on how to use this? See `OpenApiSpex.Plug.SwaggerUI` for an example.